packages/skeleton/src/lib/utilities/Popup/popup.ts

```diff
@@ -138,7 +138,7 @@ export function popup(triggerNode: HTMLElement, args: PopupSettings) {
 	}
```

Comment:
This seems out of scope for the PR. Can you confirm if this was intended. Likewise with the related changeset.